provider/manifest/http/server_test.go

```diff
@@ -37,6 +37,7 @@ func TestManifest(t *testing.T) {
 	handler.On("HandleManifest", mock.Anything).Return(nil).Once()
 
 	withServer(t, func() {
+		testutil.SleepForThreadStart(t)
```

Comment:
best to put this in `withServer` - the other test that uses it might start flapping too.